### PR TITLE
Bioimage update

### PR DESCRIPTION
**Pull request by jburel**
Adjust page following comment on https://github.com/elixir-europe/rdmkit/issues/1102

### CODE REVIEW

File: pages/your_domain/bioimaging_data.md [hunk 1]
```diff
@@ -23,19 +23,22 @@ Bioimaging specialists are acquiring an ever growing amount of data: images, ass
...
+The number of files and the size of files could be extremely large. Deleting/misplacing a file could invalidate the study itself, preventing its reuse. 
 
+Managing images immediately becomes a larger problem, not only the binary files need to be handled, but also the associated metadata. Several efforts have been made and still ongoing to capture those metadata.
+Understanding and capturing the metadata are critical for many reasons, just to mention a few: analysis, detection of possible faults in acquisition systems. It is important to decide how much details will be recorded since this could dramatically increase the metadata volume and therefore the effort required to capture the metadata.
+
+The collection of images can take several forms: 
+ - data acquired within a facility;
+ - data acquired in other facility (commissioned work or external guest user) and "transported" by the users to their facility;
```
Comment (bianchini88) [inline pages/your_domain/bioimaging_data.md:40]: It is not immediately clear why the "form" would be different depending on the facility. 
More details should be provided to further enrich this list.

File: pages/your_domain/bioimaging_data.md [hunk 2]
```diff
@@ -23,19 +23,22 @@ Bioimaging specialists are acquiring an ever growing amount of data: images, ass
...
+ - slides scanned for example.
```
Comment (bianchini88) [inline pages/your_domain/bioimaging_data.md:41]: No need to write "for example" within such a list

File: pages/your_domain/bioimaging_data.md [hunk 3]
```diff
@@ -53,26 +56,26 @@ Unlike other domains, the bioimaging community has not yet agreed on a single st
 
 ### Solutions
 
-**Vendor libraries**: Some vendors provide open source libraries for parsing their proprietary file formats. See [libCZI](https://github.com/zeiss-microscopy/libCZI) from Zeiss.
+**Vendor libraries**: Some vendors provide open source libraries for parsing their proprietary file formats. See [libCZI](https://github.com/ZEISS/libczi) from Zeiss.
 
-**Open source translators**: Members of the community have developed multi-format translators that can be used to access your data with needing to transform it. You will need to perform this translation each time you access your data.
+**Open source translators**: Members of the community have developed multi-format translators that can be used to access your data with needing to transform it. You will need to perform this translation each time you access your data. Translation libraries include,
```
Comment (bianchini88) [inline pages/your_domain/bioimaging_data.md:61]: This sentence should be rephrased to make it clearer. E.g. "to access your data with needing to transform it" is not very clear. The word "translator" is used for tools that seem to be "converters".

File: pages/your_domain/bioimaging_data.md [hunk 4]
```diff
@@ -53,26 +56,26 @@ Unlike other domains, the bioimaging community has not yet agreed on a single st
...
 
 **Permanent conversion**: An alternative is permanently convert your data to
 
-  - [OME-Files](https://www.openmicroscopy.org/ome-files/) - The [Open Microscopy Consortium (OME)](https://www.openmicroscopy.org/) has developed an open format, "OME-TIFF", to which you can convert your data. Bio-Formats (above) is likely the most straight-forward method to convert your data to OME-TIFF.
-  - The [bioformats2raw](https://github.com/glencoesoftware/bioformats2raw) and [raw2ometiff](https://github.com/glencoesoftware/raw2ometiff) toolchain provided by [Glencoe Software](https://www.glencoesoftware.com/) allows the more performant conversion of your data, but requires an extra intermediate copy of the data. If you have available space, you might want to give it a try.
+  - [OME-Files](https://www.openmicroscopy.org/ome-files/) - The [Open Microscopy Consortium (OME)](https://www.openmicroscopy.org/) has developed an open format, "OME-TIFF", to which you can convert your data. Bio-Formats (above) is likely the most straight-forward method to convert your data to OME-TIFF
```
Comment (bianchini88) [inline pages/your_domain/bioimaging_data.md:69]: "Straightforward", not "straight-forward". Also, what do the authors actually mean by "straightforward" in this case?

File: pages/your_domain/bioimaging_data.md [hunk 5]
```diff
@@ -53,26 +56,26 @@ Unlike other domains, the bioimaging community has not yet agreed on a single st
...
+  - The [bioformats2raw](https://github.com/glencoesoftware/bioformats2raw) and [raw2ometiff](https://github.com/glencoesoftware/raw2ometiff) toolchain provided by [Glencoe Software](https://www.glencoesoftware.com/) allows the more performant conversion of your data, but requires an extra intermediate copy of the data. If you have available space, you might want to give it a try
```
Comment (bianchini88) [inline pages/your_domain/bioimaging_data.md:70]: "you might want to give it a try" is possibly too colloquial.

File: pages/your_domain/bioimaging_data.md [hunk 6]
```diff
@@ -132,18 +135,18 @@ mainly due to limited infrastructures capable of hosting the data. There are a f
...
 - If you only need to make your data available online and have limited metadata associated, consider publishing in a **Data archive**.
 - If your data should be considered as a reference dataset, consider an **Added-values archive**.
 - Select and choose the repositories based on the following characteristics:
-  - Storage vs Added-value resources
-  - Images format support
+  - Storage vs Added-value resources.
+  - Images format support.
   - Supported licenses e.g. CC0 or CC-BY license. For example the [Image Data Resource (IDR)](http://idr.openmicroscopy.org/) uses Creative Commons Licenses for submitted datasets and encourages submitting authors to choose.
-  - Which types of access is required for the users e.g. download only, browse search and view data and metadata, API access, etc. 
+  - Which types of access is required for the users e.g. download only, browse search and view data and metadata, API access.
```
Comment (bianchini88) [inline pages/your_domain/bioimaging_data.md:149]: "Which type.... is" or "Which types... are"

**Review by bianchini88 — CHANGES_REQUESTED** [review]

**Comment by bedroesb** [comment]
@bianchini88 I ping you again because you are doing a great job at reviewing, tell me if I should stop doing so ;)

**Comment by bianchini88** [comment]
@bedroesb I will review this by the end of the week, possibly tomorrow.
Keep pinging me whenever required.

**Review by bianchini88 — APPROVED** [review]

**Pull request merged**